content/api/api-events.md

```diff
@@ -7,67 +7,310 @@ author: Lawrence Lane
+```
+curl -X DELETE --header 'Accept: */*' 'https://app.metricly.com/events/delete/policy/e321f5fa-a6fb-49d6-b4e6-beb468eb461f'
+```
+
+
+### Response Body
+
+The following response body has no content. The successful response code is 204.
+
+```
+No Content
+```
+
+{{% /expand %}}
+
+---
+
+## GET from /events/{eventId}
+
+{{< button href="https://app.metricly.com/swagger-ui.html#!/events/getEventUsingGET" theme="info" >}} GET {{< /button >}} Use this endpoint to obtain details about an event and its associated policy/elements.
+
+{{% expand "View Method Details." %}}
+
+### Parameters
+
+| Parameter | Parameter Type | Data Type | Description |
+|--------------------|----------------|-----------|----------------------------------------------|
+| term | path | string | The element field you want to aggregate. |
+| elasticsearchQuery | body | JSON | A JSON query. |
 
-**Body Attributes**
+### Request URL
 
-| Attribute | Required/Optional | Description |
-|-----------|-------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| type | Required | The type attribute tells CloudWisdom what type of event this is; valid inputs are INFO, WARNING, or CRITICAL. |
-| tags | Optional | You can tag events to enable filtering in CloudWisdom by passing in keys and values. The format for tags is [{“name”:”key1″, “value”:”value1″}, {“name”:”key2″, “value”:”value2″}, (etc.)] |
-| data | Required | The data attribute provides CloudWisdom with the content for the actual event. It contains three attributes: **elementId (required)**: This should contain the fully qualified name (fqn) of the element you wish to generate an external event for. **level (optional)**: The severity level of the event. Valid inputs are Info, Warning, or Critical. If values are set at both level and type, the level value will take precedence since it’s closer to the message itself and more specific. **message (optional)**: The message text generated for the event. |
-|source |optional | Source of the event. This is set to the ingested integration if the value is missing. |
-| title| required | Title of the event |
-|timestamp   | optional  | You can define a timestamp, but it must be in the epoch format. If you do not define a timestamp, CloudWisdom will create one for you.  |
+```
+https://app.metricly.com/events/{eventId}
+
+```
+
+### CURL
+
+In the following CURL example, the **eventId** `82691285-5254-4830-ba07-34b809cfe25b` is used.
```

Comment:
I don't think the event ID needs to be copyable either. Though, I do like them having a different format. Is there a way to make it a code span without the copy button?

Reply:
I'm unsure if there's another way to do codespan without it auto-generating the copy button. I like to format the value in code but yeah this happens haha.